join-badguys.zed

```diff
@@ -1,6 +1,6 @@
 from (
   demo.pcap => sort id.resp_h ;
-  BadGuys ;
+  BadGuys => sort addr ;
```

Comment:
This stood out for me since in the Support context I'm accustomed to making sure the `sort` is always present. I wasn't sure if maybe you were leveraging some advanced Zed lake trick I wasn't aware of, but sure enough once I added the sort, the record count in my follow-on `zapi query "count() by _path"` went from 17 to hundreds. However, the additional filtering with `zapi query "count() by _path | badguy"` also didn't do anything special for me because 100% of the records that made it through the `inner join` had the `badguy` field appended. So I may be missing some of the intent here and mostly want to call it to your attention...

Reply:
I meant to do an orderby addr on the Badguys but that is too subtle.  The sort works!